test/TestAssets/TestProjects/WatchNoDepsApp/WatchNoDepsApp.csproj

```diff
@@ -21,4 +21,8 @@
     <Warning Text="The value of property is '$(TestProperty)'" />
   </Target>
 
+  <ItemGroup>
```

Comment:
Is this empty ItemGroup intentional?

Reply:
Yes, makes it easier to write test that inserts items into the project file.